### PR TITLE
Update SetPedCanPlayAmbientIdles.md

### PR DESCRIPTION
**Pull request by freedy69**
Added parameter info and also description

### CODE REVIEW

**Review by PsychoShock — COMMENTED** [review]
Hello,

For better docs, can you change the params by adding `b` before to understand it's a bool.

Thanks :)

File: TASK/SetPedCanPlayAmbientIdles.md [hunk 1]
```diff
@@ -6,18 +6,14 @@ aliases: ["0x8FD89A6240813FD0"]
...
 ```
 
-```
-Appears only in fm_mission_controller and used only 3 times.  
-ped was always PLAYER_PED_ID()  
-p1 was always true  
-p2 was always true  
-```
+Prevents a ped from playing ambient idle animations.
+This native must be called every frame.
```
Comment (PsychoShock) [inline TASK/SetPedCanPlayAmbientIdles.md:13]: Can you change the description in this format:

```
Prevents a ped from playing ambient idle animations.

Note: This native must be called every frame.
```

**Comment by freedy69** [comment]
> Hello,
> 
> 
> 
> For better docs, can you change the params by adding `b` before to understand it's a bool.
> 
> 
> 
> Thanks :)

Sure but im confused because some commits have those and some dont?

File: TASK/SetPedCanPlayAmbientIdles.md (hunk 1 above)
Comment (freedy69) [reply TASK/SetPedCanPlayAmbientIdles.md:13]: To a codeblock?

**Comment by PsychoShock** [comment]
Yeah well it's suposse to be a "standart" to have every native with this following but it's hard to keep track on everyone of them. This is just my personnal opinion, maybe @4mmonium don't mind.

**Review by 4mmonium — APPROVED** [review]
Thank you for your contribution, approved!

**Pull request merged**